openedx/adg/lms/webinars/tests/test_helpers.py

```diff
@@ -66,47 +65,41 @@ def test_send_webinar_emails(mocker, template_slug):
 
 @pytest.mark.django_db
 @pytest.mark.parametrize(
-    'registered_users_with_response, cohosts, panelists, presenter, expected_email_addresses',
+    'registered_users_with_response, cohosts, panelists, expected_email_addresses',
     [
         (
-            [], [], [], 't3@eg.com', ['t3@eg.com']
+            [('t1@eg.com', True), ('t2@eg.com', False)], ['t1@eg.com'], [], ['t1@eg.com']
```

Comment:
Let's create constants for these as well.

Reply:
IMO for test cases we should not go with too much constants, these are just parameters to test different scenarios so we can use them as it is